src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputNoLineWrapBad.java

```diff
@@ -1,6 +1,9 @@
 package com.puppycrawl.tools.
     checkstyle.checks.whitespace;
 
+import static java.math.
```

Comment:
please move this import to the bottom to minimize changes in UT files.

please add comment " //violation" on line where you expect it.